### PR TITLE
set qt and h in extrap buoyancy to clear values

### PR DESCRIPTION
**Pull request by yairchn**
First step to handle the definition of h and qt in extra buoyancy part in the updrafts

### CODE REVIEW

**Review by charleskawczynski — APPROVED** [review]
I agree that this is better than before, but is there not a local fix (at `k` rather than `k-1`) that can be applied here?

Also, let's delete the local `qt` and `h` initialized above

File: src/EDMF_Updrafts.jl [hunk 1]
```diff
@@ -395,8 +393,8 @@ function buoyancy(
                     UpdVar.B.values[i, k] = buoyancy_c(self.Ref.rho0_half[k], rho)
                     UpdVar.RH.values[i, k] = relative_humidity_c(self.Ref.p0_half[k], qt, qt - qv, 0.0, t)
                 elseif UpdVar.Area.values[i, k - 1] > 0.0 && k > kc_surf
-                    # TODO: report bug:
-                    # qt and h were not defined here before the function call.
+                    qt = UpdVar.QT.values[i,k-1]
+                    h  = UpdVar.H.values[i,k-1]
                     sa = eos(self.Ref.p0_half[k], qt, h)
                     qt -= sa.ql
```
Comment (trontrytel) [inline src/EDMF_Updrafts.jl:399]: We still need to figure out this guy. Do you want to do it in the next PR?

Comment (yairchn) [reply src/EDMF_Updrafts.jl:399]: yes next up

**Pull request merged**